201-encrypt-create-new-vm-gallery-image/azuredeploy.json

```diff
@@ -52,31 +52,42 @@
                 "description": "Name of Subnet to which the VM NIC belongs to"
             }
         },
-        "AADClientID": {
+        "aadClientID": {
             "type": "string",
             "metadata": {
                 "description": "Client ID of AAD app which has permissions to KeyVault"
             }
         },
-        "AADClientSecret": {
+        "aadClientSecret": {
             "type": "securestring",
             "metadata": {
                 "description": "Client Secret of AAD app which has permissions to KeyVault"
             }
         },
-        "KeyVaultResourceID": {
+        "keyVaultResourceID": {
             "type": "string",
```

Comment:
Can't you potentially get this using resourceId function, name of the key vault and resource group of the key vault?